requirements/requirements.in

```diff
@@ -1,6 +1,6 @@
 attrs==18.1.0
-# needed to build gevent from source
-cython==0.27.3
+# used to build jsonobject from source
+cython>=0.27.3
```

Comment:
if we're bumping all the `.txt` files (in this case to `0.29.1`) does it not also make sense to bump the `.in` file? seems hard to know whether we break on e.g. `0.28` otherwise.

Reply:
(this comment applies everywhere in this file)

Reply:
I changed the requriments to use ["comaptible" syntax](https://pip.pypa.io/en/stable/reference/pip_install/#example-requirements-file)

> if we're bumping all the .txt files (in this case to 0.29.1) does it not also make sense to bump the .in file? seems hard to know whether we break on e.g. 0.28 otherwise.

I think that should be fine and the expected way forward. I'd like to get to a point where `make upgrade-requirements` can be run regularly and keep our dependencies up to date and that will update the .txt files, but not the in files. Since deploys use .txt files to install packages, we probably won't care if we break on intermittent packages, we'll only care if new packages break (which should be caught by tests and can be fixed by limiting installs with <x.y.z)

Reply:
cool, agree with your goals and that plan makes sense to me.